treeherder/model/models.py

```diff
@@ -221,6 +221,14 @@ class BugscacheOccurrence(models.Model):
     bug = models.ForeignKey("Bugscache", on_delete=models.CASCADE, related_name="occurrences")
     created = models.DateTimeField(auto_now_add=True)
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["user", "failure_line", "bug"],
```

Comment:
The condition could also be `fields=("failure_line", "bug")` - there is no value in having the same classification multiple times by different people.